cvefeed/stats_test.go

```diff
@@ -0,0 +1,211 @@
+        "references": {
+          "reference_data": [
+            {
+              "name": "",
+              "refsource": "",
+              "tags": [
+                "Vendor Advisory"
+              ],
+              "url": ""
+            },
+            {
+              "name": "test",
+              "refsource": "MISC",
+              "url": ""
+            }
+          ]
+        }
+      },
+      "configurations": {
+        "CVE_data_version": "4.0",
+        "nodes": [
+		  {
+            "children": [
+              {
+                "cpe_match": [
+                  {
+                    "cpe23Uri": "cpe:2.3:a:test:test:-:*:*:*:*:*:*:*",
+                    "vulnerable": true
+                  }
+                ],
+                "operator": "OR"
+              },
+              {
+                "cpe_match": [
+                  {
+                    "cpe23Uri": "cpe:2.3:h:test:test:-:*:*:*:*:*:*:*",
+                    "vulnerable": false
+                  }
+                ],
+                "operator": "OR"
+              }
+            ],
+            "operator": "AND"
+		  },
+		  {
+            "children": [
+              {
+                "cpe_match": [
+                  {
+                    "cpe23Uri": "cpe:2.3:a:test:test:-:*:*:*:*:*:*:*",
+                    "vulnerable": true
+                  }
+                ],
+                "operator": "OR"
+              },
+              {
+                "cpe_match": [
+                  {
+                    "cpe23Uri": "cpe:2.3:h:test:test:-:*:*:*:*:*:*:*",
+                    "vulnerable": false
+				  },
+				  {
+                    "cpe23Uri": "cpe:2.3:h:test:test:-:*:*:*:*:*:*:*",
+                    "vulnerable": false
+                  }
+                ],
+                "operator": "OR"
+              }
+            ],
+            "operator": "AND"
+          },
+          {
+            "children": [
+              {
+                "cpe_match": [
+                  {
+                    "cpe23Uri": "cpe:2.3:o:test:test:-:*:*:*:*:*:*:*",
+                    "vulnerable": true
+                  }
+                ],
+                "operator": "OR"
+              },
+              {
+                "cpe_match": [
+                  {
+                    "cpe23Uri": "cpe:2.3:a:test:test:-:*:*:*:*:*:*:*",
+                    "vulnerable": false
+                  },
+                  {
+                    "cpe23Uri": "cpe:2.3:h:test:test:-:*:*:*:*:*:*:*",
+                    "vulnerable": false
+                  }
+                ],
+                "operator": "OR"
+              }
+            ],
+            "operator": "AND"
+		  },
+		  {
+			"cpe_match": [
+			  {
+				"cpe23Uri": "cpe:2.3:a:test:test:-:*:*:*:*:*:*:*",
+				"vulnerable": true
+			  }
+			],
+			"operator": "OR"
+		  }
+        ]
+      }
+    }
+  ]
+}
+`
+
+func TestStats(t *testing.T) {
+	file, err := ioutil.TempFile("/tmp", "test_nvd.json")
```

Comment:
Note that there's a way to make testing less verbose than what the standard library provides: https://godoc.org/github.com/stretchr/testify/assert. That has become the default way I write tests because it's way too much typing otherwise :)